tests/platform_tests/counterpoll/counterpoll_helper.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+from tests.platform_tests.counterpoll.counterpoll_constants import CounterpollConstants
+
+
+class ConterpollHelper:
+    @staticmethod
+    def get_counterpoll_show_output(duthost):
+        counterpoll_show = duthost.command(CounterpollConstants.COUNTERPOLL_SHOW)
```

Comment:
Suggest use duthost.show_and_parse(), it could generate a JSON output and you don't have to parse the output again.

Reply:
fix